core/src/builtin_widgets/anchor.rs

```diff
@@ -191,7 +192,8 @@ impl WrapRender for RelativeAnchor {
       })
       .unwrap_or_default();
 
-    ctx.update_position(ctx.widget_id(), Point::new(x, y));
+    let pos = ctx.position(ctx.widget_id()).unwrap_or_default();
```

Comment:
You can replace `ctx.position(ctx.widget_id())` with `ctx.box_pos()` for a more concise expression.